pkg/grpc/api_test.go

```diff
@@ -28,6 +28,8 @@ import (
 
 	"github.com/dapr/components-contrib/lock"
 
+	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Do we really need this dependency just to define some mocks down for test?

Reply:
This import should be ok.